.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Report a bug
+about: Something with Polygon zkEVM is not working as expected
+title: ''
+labels: 'type:bug'
+assignees: ''
+---
+
+#### System information
+
+zkEVM version: `zkEVM version`
```

Comment:
this template will be used only for mainnet? if it will be used also in testnet then it will good to add a field to specify the network (mainnet/testnet). Also as zkEVM version we don't have an specific version control, we have only the name of the fruit, maybe we need to add fields to specify the version of the prover/executor, node and bridge

Reply:
Added Node to the version field name, now it´s the zkEVM Node Version, with a suggested value that matches the node version
Added the Network field, with suggested values Mainnet and Testnet